### PR TITLE
crowdstrike/fdr: Prevent ingesting documents without `@timestamp`

### PR DESCRIPTION
**Pull request by adriansr**
## What does this PR do?

In some cases, FDR documents can lack the `@timestamp` field due to
missing timestamp source fields (`timestamp`/`CreationTimestamp`).

This adds support for the `UTCTimestamp` field and also adds a
last-chance processor to set `@timestamp` to the ingest timestamp in
case all fields are missing.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### CODE REVIEW

File: packages/crowdstrike/data_stream/fdr/elasticsearch/ingest_pipeline/default.yml [hunk 1]
```diff
@@ -11,28 +11,46 @@ processors:
...
+      if: ctx.event?.created == null
   - set:
       tag: set-timestamp
       field: "@timestamp"
       copy_from: event.created
       if: ctx?.event?.created != null && (ctx?.crowdstrike?.ContextTimeStamp == null || ctx?.crowdstrike?.ContextTimeStamp == "")
+  - set:
+      tag: set-timestamp-ingest
+      field: "@timestamp"
+      value: "{{{ _ingest.timestamp }}}"
```
Comment (andrewkroh) [inline packages/crowdstrike/data_stream/fdr/elasticsearch/ingest_pipeline/default.yml:52]: Was there a reason you chose `value` with a template over a `copy_from`?

Comment (adriansr) [reply packages/crowdstrike/data_stream/fdr/elasticsearch/ingest_pipeline/default.yml]: No reason, fixed.

File: packages/crowdstrike/data_stream/fdr/_dev/test/pipeline/test-notimestamp.log-expected.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
+{
+    "expected": [
+        {
+            "@timestamp": "2022-06-08T10:59:53.840983900Z",
+            "crowdstrike": {
+                "AgentLoadFlags": "0",
+                "AgentLocalTime": "2021-11-09T05:47:19.952Z",
```
Comment (andrewkroh) [inline packages/crowdstrike/data_stream/fdr/_dev/test/pipeline/test-notimestamp.log-expected.json:7]: Did you consider using `AgentLocalTime`?

I think `Time` could also be considered (probably before the _ingest.timestamp). That is supposed to be the timestamp of when the event was received by CrowdStrike cloud.

Comment (adriansr) [reply packages/crowdstrike/data_stream/fdr/_dev/test/pipeline/test-notimestamp.log-expected.json]: Thanks. For now I used AgentLocalTime. This covers all our samples.

**Pull request merged**